### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by nicolocarpignoli** [AR-js-org/studio#43]
We should try to make this web app more accessible.

From screen reader support to keyboard navigation.

I will add a shortlist to check. 
This issue will probably be some of the last for the MVP to implement.

- [ ] Try with a screen reader and look for problems with speech synthesis, and fix
- [ ] Look for problems with keyboard navigation (missing tabindex, wrong tab order, watch out for traps) and fix
- [ ] Be sure that all important actions (such as the creation flow) are doable with keyboard and speech synthesis
- [ ] Look for color contrasts  issues (e.g. with https://a11ygator.chialab.io/)
- [ ] Look for other general a11y issues (e.g. with https://a11ygator.chialab.io/)
- [ ] Optional (but it's the most important, actually!) Look for professional users with disabilities to test accessibility of the project, even without going too much in detail.

**Comment by mingir2** [comment]
For both marker and location page, the upload buttons (technically not buttons) aren't accessible via tab.

**Comment by nicolocarpignoli** [comment]
That's not good, we should use real buttons

**Comment by nicolocarpignoli** [comment]
We will see on the final MVP phase

**Comment by gciandro13** [comment]
Ok, i will take a look at it next week with https://a11ygator.chialab.io/

**Comment by gciandro13** [comment]
I will take into account those two micro-issues:

- Look for color contrasts issues (e.g. with https://a11ygator.chialab.io/)
-  Look for other general a11y issues (e.g. with https://a11ygator.chialab.io/)

**Comment by gciandro13** [comment]
https://github.com/AR-js-org/studio/pull/84 
Allygator error issues closed.

**Comment by nicolocarpignoli** [comment]
@gciandro13 please check the solved points on the checklist on first post.
If you'd like to see how AR.js Studio behaves with keyboard navigation, you can try using VoiceOver (with a Macbook).

Most important things to check is the possibility to navigate alle the UI elements and that all buttons/inputs are usable from keyboard. If not, it means that we have not used standard elements.
